Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+﻿<Project>
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' != 'netstandard2.0' and !$(TargetFramework.StartsWith('netcoreapp2')) and !$(TargetFramework.StartsWith('net4'))">
```

Comment:
adding `net6` or any other tfm in the future would include `HAS_ASYNC_ENUMERATOR` without anyone having to remember opting-in.